tests/test_sagemaker/test_sagemaker_feature_groups.py

```diff
@@ -0,0 +1,90 @@
+"""Unit tests for sagemaker-supported APIs."""
+from datetime import datetime
+
+import boto3
+from freezegun import freeze_time
+
+from moto import mock_sagemaker
+
+# See our Development Tips on writing tests for hints on how to write good tests:
+# http://docs.getmoto.org/en/latest/docs/contributing/development_tips/tests.html
+
+
+@mock_sagemaker
+def test_create_feature_group():
+    client = boto3.client("sagemaker", region_name="us-east-2")
+    resp = client.create_feature_group(
+        FeatureGroupName="some-feature-group-name",
+        RecordIdentifierFeatureName="some_record_identifier",
+        EventTimeFeatureName="EventTime",
+        FeatureDefinitions=[
+            {"FeatureName": "some_feature", "FeatureType": "String"},
+            {"FeatureName": "EventTime", "FeatureType": "Fractional"},
+            {"FeatureName": "some_record_identifier", "FeatureType": "String"},
+        ],
+        RoleArn="arn:aws:iam::123456789012:role/AWSFeatureStoreAccess",
+        OfflineStoreConfig={
+            "DisableGlueTableCreation": False,
+            "S3StorageConfig": {"S3Uri": "s3://mybucket"},
+        },
+    )
+
+    assert (
+        resp["FeatureGroupArn"]
+        == "arn:aws:sagemaker:us-east-2:123456789012:feature-group/some-feature-group-name"
+    )
+
+
+@freeze_time("2024-01-17 17:27:37")
```

Comment:
Freezetime doesn't work in ServerMode, so this would fail atm.

Considering the exact value of the creation date doesn't matter, I would prefer to just check if the value exists:
```
assert isinstance(resp["CreationDate"], datetime)
```
That way we don't have to mock the date, and it will work regardless of the exact date/format.

Reply:
Yes, I see it done ;)